### PR TITLE
fix/pcd verifier interface

### PR DESCRIPTION
**Pull request by vuvoth**
## Motivation

PCD verifier use 'hard-code' URL, We should push it to parameters. 

## Change Summary

Change verifier interface and update test

## Merge Checklist

_Choose all relevant options below by adding an `x` now or at any time before submitting for review_

- [ ] PR title adheres to the [conventional commits](https://www.conventionalcommits.org/en/v1.0.0/) standard
- [ ] PR has a [change set](https://github.com/privacy-scaling-explorations/anon-aadhaar/CHANGELOG.md)
- [ ] PR has been tagged with a change label(s) (i.e. documentation, feature, bug fix, or chore)
- [ ] PR includes documentation if necessary.

## Additional Context

If this is a relatively large or complex change, provide more details here that will help reviewers

### CODE REVIEW

File: packages/anon-aadhaar-pcd/src/pcd.ts [hunk 1]
```diff
@@ -45,7 +46,7 @@ export async function init(args: PCDInitArgs): Promise<void> {
 export async function prove(args: IdentityPCDArgs): Promise<IdentityPCD> {
   if (!initArgs) {
     throw new Error(
-      'cannot make semaphore signature proof: init has not been called yet'
+      'cannot make Country Identity proof for aadhaar card: init has not been called yet'
```
Comment (Meyanis95) [inline packages/anon-aadhaar-pcd/src/pcd.ts:49]: I think we can get rid of the name "country identity" as discussed

'cannot make Anon Aadhaar proof: init has not been called yet'

**Review by Meyanis95 — CHANGES_REQUESTED** [review]
Thanks!!

I just wanted to have more insights on why this change, have you spotted some potential bugs? 
Because these constant need to be hardcoded somewhere else

Don't forget to also modify the verify function call in `anon-aadhaar-react` => https://github.com/privacy-scaling-explorations/anon-aadhaar/blob/6f37da20af50552db6525ebc798981e37416014f/packages/anon-aadhaar-react/src/provider/AnonAadhaarProvider.tsx#L217

Also What do you think of having a shared "constants" folder to manage these variables?

File: packages/anon-aadhaar-pcd/src/pcd.ts (hunk 1 above)
Comment (vuvoth) [inline packages/anon-aadhaar-pcd/src/pcd.ts:49]:
```suggestion
      'cannot make Anon Aadhaar proof: init has not been called yet'
```

**Comment by vuvoth** [comment]
> I just wanted to have more insights on why this change, have you spotted some potential bugs?

It not really is a bug I think, I suggest change the verifier interface just for create more flexible for our SDK. When we hardcored a verify key as constant this make developers hard to make a env for testing, debug and extend. 

More ever,  If we support more country identity this key will change for each country, I think push it to parameters help developers easy to control logic in source code.

File: packages/anon-aadhaar-pcd/src/pcd.ts [hunk 2]
```diff
@@ -72,26 +72,28 @@ export async function prove(args: IdentityPCDArgs): Promise<IdentityPCD> {
...
-      .get('https://d3dxq5smiosdl4.cloudfront.net/verification_key.json')
-      .then(response => {
-        return response.data
-      })
+  if (!initArgs) {
+    throw new Error(
+      'cannot make Anon Aadhaar proof: init has not been called yet'
+    )
+  }
+  console.log(vk)
```
Comment (Meyanis95) [inline packages/anon-aadhaar-pcd/src/pcd.ts:82]: You forgot a console.log

**Review by Meyanis95 — APPROVED** [review]

**Pull request merged**